app/src/main/java/org/oppia/android/app/walkthrough/topiclist/topiclistviewmodel/WalkthroughTopicHeaderViewModel.kt

```diff
@@ -1,7 +1,16 @@
 package org.oppia.android.app.walkthrough.topiclist.topiclistviewmodel
 
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
 import org.oppia.android.app.walkthrough.topiclist.WalkthroughTopicItemViewModel
 
 /** [ViewModel] What do want to learn text in [WalkthroughTopicListFragment]. */
-class WalkthroughTopicHeaderViewModel : WalkthroughTopicItemViewModel()
+class WalkthroughTopicHeaderViewModel : WalkthroughTopicItemViewModel() {
+  private val isHeaderTextVisibleMutableLiveData = MutableLiveData<Boolean>(true)
+  val isHeaderTextVisible: LiveData<Boolean> = isHeaderTextVisibleMutableLiveData
```

Comment:
@anandwana001 @rt4914 
I am trying to use a similar approach to hide this inner text but I am not able to use this `viewModel` in `activityPresenter` or `fragmentPresenter` because we don't have a dependency of `ViewModelProvider<WalkthroughTopicItemViewModel>` or `ViewModelProvider<WalkthroughTopicHeaderViewModel>`
Could you please help?

Reply:
Suggest using a listener to send information to Fragment/Activity , which once triggered, you can use to hide/show view.